recipes/openms-meta/meta.yaml

```diff
@@ -141,13 +141,10 @@ outputs:
         - {{ pin_compatible('sqlite', max_pin='x.x') }}
     test:
       commands:
-        - OPENMS_DATA_PATH=${PREFIX}/share/OpenMS/ ${PREFIX}/bin/FileMerger --help
+        - OPENMS_DATA_PATH=${PREFIX}/share/OpenMS/ ${PREFIX}/bin/OpenMSInfo
```

Comment:
why is this change needed? Can you add both?

Reply:
Yes sure. Can be both. OpenMSInfo just gives some info about the build, too, to see if the settings were applied correctly.

Reply:
If the version information is in the output of `OpenMSInfo` you could try to match it with the package version.

Reply:
I guess both is a good idea.